lightningd/peer_control.h

```diff
@@ -73,6 +73,7 @@ struct peer *find_peer_by_dbid(struct lightningd *ld, u64 dbid);
 struct peer *new_peer(struct lightningd *ld, u64 dbid,
 		      const struct node_id *id,
 		      const struct wireaddr_internal *addr,
+			  const u8 *their_features,
```

Comment:
same here

Reply:
the other lines have 4 space tabs I contend :)